tests/behaviour/context.py

```diff
@@ -17,21 +17,38 @@
 # under the License.
 #
 from concurrent.futures._base import Future
-from typing import List, Union
+from typing import List, Union, Optional
 
 import behave.runner
+from behave.model import Table
 
 from grakn.client import GraknClient
+from grakn.concept.answer.concept_map import ConceptMap
+from grakn.concept.answer.concept_map_group import ConceptMapGroup
+from grakn.concept.answer.numeric import Numeric
+from grakn.concept.answer.numeric_group import NumericGroup
+from grakn.concept.concept import Concept
 from grakn.concept.thing.attribute import Attribute, BooleanAttribute, LongAttribute, DoubleAttribute, StringAttribute, \
     DateTimeAttribute
 from grakn.concept.thing.entity import Entity
 from grakn.concept.thing.relation import Relation
 from grakn.concept.thing.thing import Thing
+from grakn.concept.type.attribute_type import AttributeType, BooleanAttributeType, LongAttributeType, \
+    DoubleAttributeType, StringAttributeType, DateTimeAttributeType
+from grakn.concept.type.entity_type import EntityType
+from grakn.concept.type.relation_type import RelationType
+from grakn.concept.type.role_type import RoleType
+from grakn.concept.type.thing_type import ThingType
+from grakn.concept.type.type import Type
 from grakn.rpc.session import Session
 from grakn.rpc.transaction import Transaction
 
 
-ThingSubtype: Thing = Union[Entity, Relation, BooleanAttribute, LongAttribute, DoubleAttribute, StringAttribute, DateTimeAttribute]
+AttributeSubtype: Attribute = Union[BooleanAttribute, LongAttribute, DoubleAttribute, StringAttribute, DateTimeAttribute]
+ThingSubtype: Thing = Union[Entity, Relation, AttributeSubtype]
+TypeSubtype: Type = Union[ThingType, EntityType, RelationType, RoleType, AttributeType, BooleanAttributeType, LongAttributeType, DoubleAttributeType, StringAttributeType, DateTimeAttributeType]
+ConceptSubtype: Concept = Union[ThingSubtype, TypeSubtype]
```

Comment:
I looked for a more elegant way of expressing these, and tried `typing.Type`- but the trouble with that is that it includes the type itself that you specify, so it was thinking, for example, that `ConceptSubtype` could be a `Concept`, which is untrue.